apps/api/src/app.module.ts

```diff
@@ -59,12 +59,6 @@ const enterpriseImports = (): Array<Type | DynamicModule | Promise<DynamicModule
     }
   }
 
-  if (isClerkEnabled()) {
-    if (require('@novu/ee-auth')?.EEAuthModule) {
-      modules.push(require('@novu/ee-auth')?.EEAuthModule);
-    }
-  }
-
```

Comment:
This is not needed here because, we always import `AuthModule` which implementation dynamically changes to either EE or community inside the actual module